script.js

```diff
@@ -1,12 +1,208 @@
+const selectShows = document.querySelector("#select-shows");
+// Select related to episodes
+const selectEpisodes = document.querySelector("#select-episodes");
+let allShows;
+let allEpisodes;
+
+//EventListeners
+// display episodes on load
+document.addEventListener("DOMContentLoaded", setup);
+// Search specific Show based on select
+selectShows.addEventListener("change", selectShow);
+// Search specific episode based on select
+selectEpisodes.addEventListener("change", selectEpisode);
+// Search episodes
+searchUser.addEventListener("keyup", searchMovie);
+
+//Functions
+// Get all shows
+async function setup() {
+  //get all shows with fetch
+  allShows = await getAllAvailablShows();
+  // Make card for all shows and create options for select
+  makeCardForShows(allShows);
+}
+
+//Get all shows via fetch
+async function getAllAvailablShows() {
+  // await response of the fetch call
+  const response = await fetch("https://api.tvmaze.com/shows");
+  // only proceed once its resolved
+  const data = await response.json();
+  // only proceed once second promise is resolved
+  return data;
 }
 
-function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+// Make card for all shows
+function makeCardForShows(allShows) {
+  searchUser.disabled = true;
+  allShows.sort((a, b) =>
+    a.name.toLowerCase().localeCompare(b.name.toLowerCase())
+  );
+  let output = "";
+  allShows.forEach((show) => {
+    output += `
+      <div class="col" style="height: 550px">
+        <div class="card h-100 px-0 mx-1">
+          <img src="${
+            show.image !== null ? show.image.medium : ""
+          }" style="height: 250px" class="card-img-top mx-0" alt="...">
+          <div class="card-body mb-4">
+          <h5 class="text-primary">${show.name}</h5>
+          <span class="badge text-bg-dark">Rated: ${show.rating.average}</span>
+          <span class="badge text-bg-dark">Genres: ${show.genres}</span>
+          <span class="badge text-bg-dark">Status: ${show.status}</span>
+          <span class="badge text-bg-dark">Runtime: ${show.runtime}</span>
+          <p>${show.summary}</p>
+          </div>
+        </div>
+      </div>
+    `;
+  });
+  showEpisodseRow.innerHTML = output;
+
+  // Create option for each show in select after loading page
+  allShows.forEach((show) => {
+    const option = document.createElement("option");
+    option.value = show.name;
+    option.textContent = show.name;
+    selectShows.appendChild(option);
+  });
+}
+
+// Search specific Shows based on select
+async function selectShow(event) {
+  if (event.target.value == "allSerials") {
+    setup();
+    //Remove options from episodes's select
+    const options = selectEpisodes.getElementsByTagName("option");
+    for (let i = options.length - 1; i > 0; i--) {
+      selectEpisodes.removeChild(options[i]);
+    }
+    searchUser.value = "";
```

Comment:
It looks like you're clearing out the value of searchUser in both branches of this `if` - whether the user has selected "allSerials" or one specific one. I'd recommend moving this line of code to outside the `if` statement (either before or after it), both to make more clear to the reader of code that this happens unconditionally, and to avoid repetition.

The same is true for your deleting of all of the episode options.

Reply:
I've done it 🙏